src/app/app-routing.module.ts

```diff
@@ -3,7 +3,6 @@ import { SignUpComponent } from './component/user/auth/sign-up/sign-up.component
 import { AuthComponent } from './component/user/auth/auth.component';
```

Comment:
general: please move map directory to the root level(the same level as eco-news directory)

Reply:
fixed

Reply:
fixed